.ci/override-tekton-bundle.sh

```diff
@@ -1,89 +0,0 @@
-#!/bin/bash
```

Comment:
this script was rewritten and moved to https://github.com/redhat-appstudio/e2e-tests/pull/261